extensions/objects/templates/sanitized-url-editor.component.ts

```diff
@@ -77,6 +81,16 @@ export class SanitizedUrlEditorComponent {
     this.valueChanged.emit(this.value);
     this.changeDetectorRef.detectChanges();
   }
+
+  ngAfterViewInit(): void {
+    this.form.statusChanges.subscribe((validationStatus) => {
```

Comment:
Where are the signals to this subscription coming from? (What was behaving wrongly before that caused it to be added?)

Reply:
These (signals) are coming from the schema-based-editor in the component html file. We add a `@ViewChild('schemaBasedEditorForm') form!: NgForm;` above, in line (approximately) 50, to get the instance of NgForm in the component. This form has one control which is the schema-based-editor. Hence, whenever the validation on that schema-based-editor changes, it will bubble up the error to object editor and object editor to its parent. Previously, validation was not getting passed on to the object editor up above.